api/views/user_views.py

```diff
@@ -249,9 +249,11 @@ class UploadAudioView(APIView):
     @transaction.atomic
     def post(self, request, *args, **kwargs):
         instructor = get_object_or_404(Instructor, user=request.user)
-
+        title = ""
+        if "title" in request.data:
+            title = request.data["title"]
```

Comment:
All of this can be replaced with: `title = request.data.get("title", "")`

Reply:
Good to know, fixing that now.